pipelines/install/02-install.yaml

```diff
@@ -0,0 +1,12 @@
+---
+- name: install server
+  hosts:
+   - "{{ forklift_server_name }}"
+  become: yes
```

Comment:
We apply become on our specific tasks. You shouldn't set it for the entire role/installer (although many users do that.)